tests/Behat/DrupalContext.php

```diff
@@ -54,6 +54,9 @@ public static function uninstallTestModule(AfterSuiteScope $scope): void {
    */
   public function setupDemo(BeforeScenarioScope $scope): void {
     \Drupal::service('module_installer')->install(['oe_theme_demo']);
+    $theme_name = \Drupal::theme()->getActiveTheme()->getName();
+    \Drupal::configFactory()->getEditable($theme_name . '.settings')
+      ->set('branding', 'standardised')->save();
```

Comment:
```suggestion
      ->set('branding', 'core')->save();
```
This should be `core` right? it is here below.

Reply:
It is a special case. In the Demo feature, we are testing also menu functionality but available only in standards banding.